assets/src/edit-story/components/library/panes/pageTemplates/templateList.js

```diff
@@ -84,14 +84,16 @@ function TemplateList({ pages, parentRef, pageSize }) {
   });
 
   const handleKeyboardPageClick = useCallback(
-    ({ key }, page) => {
-      if (key === 'Enter') {
-        if (isGridFocused) {
+    ({ key, code }, page) => {
+      if (isGridFocused) {
+        if (key === 'Enter') {
           handlePageClick(page);
+        } else if (code === 'Space') {
+          handleDelete?.(page);
         }
       }
     },
```

Comment:
I actually think both should be using `KeyEvent.code`. The code for <kbd>Enter</kbd> is (surprise!) `"Enter"`.